charts/tfy-karpenter-config/values.yaml

```diff
@@ -222,3 +222,40 @@ karpenter:
       else
           echo "CONTAINERD_VERSION is empty or not within the specified range."
       fi
+
+  ## provisioner for inferentia workloads
+  inferentiaProvisionerSpec:
+    ## @param karpenter.inferentiaProvisionerSpec.enabled Enable inferentia provisioner for inferentia nodes
+    ##
+    enabled: false
+
+    ## @param karpenter.inferentiaProvisionerSpec.consolidation.enabled Enable consolidation for inferentia provisioner
+    ##
+    consolidation:
+      enabled: false
+
+    ## @param karpenter.inferentiaProvisionerSpec.ttlSecondsAfterEmpty Seconds after which node should be deleted once it is empty. Either one of consolidation or ttlSecondsAfterEmpty should be used
+    ## 
+    ttlSecondsAfterEmpty: 300
+
+    ## @param karpenter.inferentiaProvisionerSpec.capacityTypes Capacity types for inferentia provisioner
+    ##    
+    capacityTypes:
+    - spot
```

Comment:
We really need to have a linter with rule on how to indent yaml lists 🥲

Reply:
I suggest we add a pre-commit hook.